webapp/channels/src/packages/mattermost-redux/src/selectors/entities/common.ts

```diff
@@ -71,10 +62,10 @@ export function getUsers(state: GlobalState): IDMappedObjects<UserProfile> {
 
 // Calls
 
-export function getProfilesInCalls(state: GlobalState): Record<string, Record<string, UserProfile>> {
+export function getSessionsInCalls(state: GlobalState): Record<string, Record<string, UserSessionState>> {
```

Comment:
Could you move both of these selectors out of `mattermost-redux` and into `selectors/calls`?

Having them in here will make it difficult whenever we get around to shipping `mattermost-redux` again as a separate package, and this file in particular is supposed to only have selectors that are reused by a lot of other selectors because import loops in `selectors/entities` cause `createSelector` to break